src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "ORDERS")
+public class Order {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "ORDER_ID", unique = true)
+    private Long orderId;
+
+    @ManyToOne
+    @JoinColumn(name = "USER_ID", nullable = false)
+    private User user;
+
+    @NotNull
```

Comment:
A czy @NotNull i nullable = false się nie duplikują tutaj? Ustalmy może której opcji używamy. Ja jestem za @NotNull :)

Reply:
Faktycznie, nie zauważyłam tego. Sprawdziłam i faktycznie w internecie piszą, że lepiej używać adnotacji @NotNull zamiast nullable. Jakby ktoś chciał poczytać to tutaj sprawdzałam:
https://www.baeldung.com/hibernate-notnull-vs-nullable